README.adoc

```diff
@@ -72,16 +72,23 @@ include::finish/src/test/java/it/io/openliberty/guides/inventory/InventoryArquil
 ----
 
 Notice that the JUnit Arquillian runner runs the tests instead of the standard JUnit runner.
-The [hotspot=RunWith]`@RunWith` annotation preceding the class tells JUnit to run the tests by using Arquillian.
+The [hotspot=RunWith file=0]`@RunWith` annotation preceding the class tells JUnit to run the tests by using Arquillian.
 
-The method annotated by [hotspot=Deployment]`@Deployment` defines the content of the
+The method annotated by [hotspot=Deployment file=0]`@Deployment` defines the content of the
 web archive, which is going to be deployed onto the Open Liberty server.
 The tests are either run on or against the server.
-The [hotspot=Testable]`testable = true` attribute enables the deployment to
+The [hotspot=Testable file=0]`testable = true` attribute enables the deployment to
 run the tests "in container", that is the tests are run on the server.
 
-Notice the [hotspot=warName hotspot=WebArchive]`arquillian-managed.war` name that is used for the web
-archive. This name is necessary if you don't want a randomly generated web archive name.
+pom.xml
+[source, xml, linenums, indent=0, role='code_column']
+----
+include::finish/pom.xml[]
+----
+
+Notice the [hotspot=warName hotspot=WebArchive file=0]`WARNAME` war file name that is used for the web
+archive and is defined in the [hotspot=ArquillianWarName file=1]`pom.xml` file.
```

Comment:
After reading this again, I think it could potentially be a bit confusing for the readers. Maybe a wording like:

> The `WARNAME` variable is used to name the web archive and is defined in the `pom.xml` file.

Reply:
@AustinSeto I updated